quesma/clickhouse/table_discovery.go

```diff
@@ -648,16 +648,10 @@ func (td *tableDiscovery) tableComment(database, table string) (comment string)
 }
 
 func (td *tableDiscovery) createTableQuery(database, table string) (ddl string) {
-	rows, err := td.dbConnPool.Query(context.Background(), "SELECT create_table_query FROM system.tables WHERE database = ? and table = ? ", database, table)
+	err := td.dbConnPool.QueryRow(context.Background(), "SELECT create_table_query FROM system.tables WHERE database = ? and table = ? ", database, table).Scan(&ddl)
```

Comment:
Is it fix? or just API call change?

Reply:
Nono, the actual error would've been in the `.Scan` part I removed - it would throw `Scan called without calling Next`.